### ISSUE TITLE
[Suggestion] Modded seed compatibility

### ISSUE THREAD

**Issue by mmelvin0** [capnkirok/animania#472]
I realize this is low-priority, but I didn't see an open or closed issue for this...

I was going to try and write a quick mod to make BWM hemp seeds placeable, but I took a look and saw that vanilla seeds are hardcoded in at least three spots so I gave up.

It may not be worth it to make it work (e.g. dynamic textures) but just figure there's a lot of mod seeds and it would be a nice to have integration at some point in the future.

**Comment by capnkirok** [comment]
Yeah, it's a good idea. I think we could add this to the config / seed placement code at some point.
